doc/BUILD_ON_DOCKER.md

```diff
@@ -0,0 +1,40 @@
+# Steps:
+
+## Step 1: clone the repository
+
+```{bash}
+git clone https://github.com/AppFlowy-IO/appflowy.git
+```
+
+## Step 2: cd into docker-buildfiles
+
+```{bash}
+cd ./appflowy/doc/docker-buildfiles
+```
+
+## Step 3: Provide access of appflowy to X session
+
+```{bash}
+xhost local:root
+```
+
+## Step 4: build the docker image
+
+```{bash}
+docker-compose build
+```
+
+## Step 5: run the docker container
+
+```{bash}
+docker-compose up
+```
```

Comment:
The application started, but only a completely white screen appeared. Here is the only output that I got:

```
➜ sudo docker-compose up
[+] Running 1/0
 ⠿ Container docker-buildfiles-app-1  Created                                                                                                                                            0.0s
Attaching to docker-buildfiles-app-1
docker-buildfiles-app-1  | 
docker-buildfiles-app-1  | ** (app_flowy:1): WARNING **: 04:05:32.084: Failed to start Flutter renderer: No available configurations for the given RGBA pixel format
```

Reply:
Are you using a cloud server for docker and exporting display from there?
Works here for me on my machine. 
![image](https://user-images.githubusercontent.com/6279035/147266023-1fdf2ca1-28a0-4570-b955-72a1f2a488b1.png)

But if I ssh to my machine from my laptop and follow the procedure then I get your error

Reply:
Nope, nothing fancy. 
Local machine (Manjaro, up to date)
Followed your script to the letter (copied pasted the instructions)
I'm glad you can at least recreate it on your laptop, it shows that I'm not a complete fool! :D

Reply:
No no, I can't do it on my laptop because I don't have enough storage. So I usually ssh my pc from my laptop. The screenshot is by connecting my pc through remote desktop.

Reply:
maybe we can ask random people on discord to try and check % of users to whom it works

Reply:
Well, they won't see this until it is merged.

So what we'll do is, I'll merge it once we're happy with it, then I'll create an issue for my white screen. If more people +1 my issue, then you can look at it.  Personally, I'm all setup, so I won't be using this feature... So it's not a problem if I'm the only one who sees that white screen...